medusa/providers/generic_provider.py

```diff
@@ -574,12 +574,25 @@ def parse_pubdate(pubdate, human_time=False, timezone=None, **kwargs):
                         matched_time = int(round(float(matched_time.strip())))
 
                     seconds = parse('{0} {1}'.format(matched_time, matched_granularity))
+                    if seconds is None:
+                        log.warning('Failed parsing human time: {0} {1}', matched_time, matched_granularity)
```

Comment:
I don't think the warning is needed since it's already going to cause an exception.

Reply:
Yeah but up until now we often had to guess why

Reply:
Yes but now the exception will include the details you added to the warning log, so you don't really need the log?

Reply:
The exeception details are not logged down the stack.

Reply:
Oh, I see now. Sorry I missed that.